### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by ckvvanle**
chapter_6

### CODE REVIEW

**Comment by ckvvanle** [comment]
ready

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
```
Comment (nvtanh) [inline app/assets/javascripts/users.coffee]: please remove files not use

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```
Comment (nvtanh) [inline app/assets/stylesheets/custom.scss]: css, js using ' replace "

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{email.downcase!}
+  validates :name, presence: true, length: {maximum: 50}
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: {maximum: 255},
+                    format: {with: VALID_EMAIL_REGEX},
+                    uniqueness: {case_sensitive: false}
+  has_secure_password
+  validates :password, presence: true, length: {minimum: 6}
```
Comment (nvtanh) [inline app/models/user.rb]: research gem "config" and apply it in here

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <%= link_to "Ruby on Rails Tutorial", "http://www.railstutorial.org/" %>
+    by <%= link_to "Michael Hartl", "http://www.michaelhartl.com/" %>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About", about_path %></li>
```
Comment (nvtanh) [inline app/views/layouts/_footer.html.erb]: research I18n and apply it in here

**Comment by nvtanh** [comment]
commented

**Comment by ckvvanle** [comment]
chapter_6

**Comment by ckvvanle** [comment]
ready

**Comment by ckvvanle** [comment]
chapter_6

**Comment by ckvvanle** [comment]
ready

**Review by hhhhuy112 — APPROVED** [review]

**Comment by hhhhuy112** [comment]
merged
You can merge it

**Pull request merged**